wiki/Community/Discord_servers/en.md

```diff
@@ -1,9 +1,105 @@
+---
+tags:
+  - development
+  - osu!dev
+---
+
```

Comment:
I see they were on the previous article but I don't think either of these should be trying to overtake searches for what are generic words for "developer". you'll find this article just fine based on its title and content. so probs remove (github won't let me write a proper suggestion for this for some reason)

Reply:
Originally the Index didn't have them. Added as this was also technically a replacement for the osu! Discord article. If they are not necessary though given users can search for page content, we'll remove them.